zoo-css-challenge/style.css

```diff
@@ -106,8 +120,16 @@ footer {
 .image-container img {
```

Comment:
Check out the `:nth-child` pseudo class as a way to change the border colour of each bear image. https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child